### PR TITLE
Portfolio published in Github Pages

### PR DESCRIPTION
**Pull request by HansZizold**
Portfolio published in Github Pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,16 @@ Github Desktop application
 
 HISTORY OF CHANGES
 Portfolio Mobile Version: finished on April 22th.
```
Comment (topeogunleye) [inline README.md:12]: - [ ] [REQUIRED] Kindly deploy your website with GitHub pages and add the live demo link to your README file.

```
## Live Demo

[Live-Demo](https://HansZizold.github.io/Portfolio/)
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @HansZizold,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [x]  Please check my inline comments.
> 
> ## [Optional] suggestions:
>- [x] Please kindly ensure that you make your README file more professional by following the microverse [template](https://github.com/microverseinc/readme-template). Here's a demo on how to copy the Microverse Template:

https://user-images.githubusercontent.com/52934128/166295474-cefabfbc-bc3b-40d7-8278-296c738ab415.mp4


> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (HansZizold) [reply README.md:13]: OK

**Review by KDlamini — APPROVED** [review]
### Hi @HansZizold 🙋‍♀️,  

Good work! 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**